src/Session.php

```diff
@@ -1233,7 +1233,7 @@ public static function haveRight($module, $right)
         }
 
         if (isset($_SESSION["glpiactiveprofile"][$module])) {
-            return intval($_SESSION["glpiactiveprofile"][$module]) & $right;
+            return intval($_SESSION["glpiactiveprofile"][$module]) && $right;
```

Comment:
This is a bitwise operation, so `&` operator is the one to use here.

Reply:
I'm almost sure such a fix will open access for profiles which should be denied. As requested, please explain the initial issue and the way to reproduce it.